templates/paragraphs/paragraph--localgov-link.html.twig

```diff
@@ -42,10 +42,11 @@
 {%
   set classes = [
     'link',
+    paragraph.localgov_link_style.value ? 'link--' ~ paragraph.localgov_link_style.value ~ 'style',
```

Comment:
Just a tiny item here. The CSS class is `.link--button-style` so we'll need a `-` before the word style.
Change 
`paragraph.localgov_link_style.value ? 'link--' ~ paragraph.localgov_link_style.value ~ 'style'` to
`paragraph.localgov_link_style.value ? 'link--' ~ paragraph.localgov_link_style.value ~ '-style'`

Reply:
That makes sense. Thank you!